src/utils/dom.js

```diff
@@ -12,7 +12,7 @@ const createElement = (elementName, attributes = {}, children) => {
   const element = document.createElement(elementName);
 
   for (let key in attributes) {
-    if (attributes[key]) {
+    if (Object.prototype.hasOwnProperty.call(attributes, key)) {
```

Comment:
is Object.keys a simpler solution?

Reply:
Object.keys return an array of keys..  how can I use it?